{"error":{},"level":"error","message":"uncaughtException: new error\nError: new error\n    at Object.<anonymous> (G:\\2021-solution-challenge\\seek\\server.js:14:7)\n    at Module._compile (node:internal/modules/cjs/loader:1108:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)\n    at Module.load (node:internal/modules/cjs/loader:973:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:813:14)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)\n    at node:internal/main/run_main_module:17:47","stack":"Error: new error\n    at Object.<anonymous> (G:\\2021-solution-challenge\\seek\\server.js:14:7)\n    at Module._compile (node:internal/modules/cjs/loader:1108:14)\n    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1137:10)\n    at Module.load (node:internal/modules/cjs/loader:973:32)\n    at Function.Module._load (node:internal/modules/cjs/loader:813:14)\n    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:76:12)\n    at node:internal/main/run_main_module:17:47","exception":true,"date":"Sat Feb 06 2021 22:11:11 GMT+0530 (India Standard Time)","process":{"pid":23392,"uid":null,"gid":null,"cwd":"G:\\2021-solution-challenge\\seek","execPath":"C:\\Program Files\\nodejs\\node.exe","version":"v15.8.0","argv":["C:\\Program Files\\nodejs\\node.exe","G:\\2021-solution-challenge\\seek\\server.js"],"memoryUsage":{"rss":64483328,"heapTotal":38326272,"heapUsed":26531848,"external":20255149,"arrayBuffers":19198648}},"os":{"loadavg":[0,0,0],"uptime":37748},"trace":[{"column":7,"file":"G:\\2021-solution-challenge\\seek\\server.js","function":null,"line":14,"method":null,"native":false},{"column":14,"file":"node:internal/modules/cjs/loader","function":"Module._compile","line":1108,"method":"_compile","native":false},{"column":10,"file":"node:internal/modules/cjs/loader","function":"Module._extensions..js","line":1137,"method":".js","native":false},{"column":32,"file":"node:internal/modules/cjs/loader","function":"Module.load","line":973,"method":"load","native":false},{"column":14,"file":"node:internal/modules/cjs/loader","function":"Module._load","line":813,"method":"_load","native":false},{"column":12,"file":"node:internal/modules/run_main","function":"Function.executeUserEntryPoint [as runMain]","line":76,"method":"executeUserEntryPoint [as runMain]","native":false},{"column":47,"file":"node:internal/main/run_main_module","function":null,"line":17,"method":null,"native":false}]}
{"error":{"name":"MongoNetworkError"},"level":"error","message":"uncaughtException: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1137:16) {\n  name: 'MongoNetworkError'\n}]\nMongoNetworkError: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1137:16) {\n  name: 'MongoNetworkError'\n}]\n    at Pool.<anonymous> (G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\topologies\\server.js:438:11)\n    at Pool.emit (node:events:378:20)\n    at G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\pool.js:562:14\n    at G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\pool.js:995:11\n    at G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\connect.js:32:7\n    at callback (G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\connect.js:280:5)\n    at Socket.<anonymous> (G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\connect.js:310:7)\n    at Object.onceWrapper (node:events:485:26)\n    at Socket.emit (node:events:378:20)\n    at emitErrorNT (node:internal/streams/destroy:188:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:153:3)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)","stack":"MongoNetworkError: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1137:16) {\n  name: 'MongoNetworkError'\n}]\n    at Pool.<anonymous> (G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\topologies\\server.js:438:11)\n    at Pool.emit (node:events:378:20)\n    at G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\pool.js:562:14\n    at G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\pool.js:995:11\n    at G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\connect.js:32:7\n    at callback (G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\connect.js:280:5)\n    at Socket.<anonymous> (G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\connect.js:310:7)\n    at Object.onceWrapper (node:events:485:26)\n    at Socket.emit (node:events:378:20)\n    at emitErrorNT (node:internal/streams/destroy:188:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:153:3)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)","exception":true,"date":"Sat Feb 06 2021 22:12:31 GMT+0530 (India Standard Time)","process":{"pid":22652,"uid":null,"gid":null,"cwd":"G:\\2021-solution-challenge\\seek","execPath":"C:\\Program Files\\nodejs\\node.exe","version":"v15.8.0","argv":["C:\\Program Files\\nodejs\\node.exe","G:\\2021-solution-challenge\\seek"],"memoryUsage":{"rss":67129344,"heapTotal":57987072,"heapUsed":24074816,"external":20246808,"arrayBuffers":18313814}},"os":{"loadavg":[0,0,0],"uptime":37828},"trace":[{"column":16,"file":"node:net","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1137,"method":"afterConnect [as oncomplete]","native":false},{"column":11,"file":"G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\topologies\\server.js","function":null,"line":438,"method":null,"native":false},{"column":20,"file":"node:events","function":"Pool.emit","line":378,"method":"emit","native":false},{"column":14,"file":"G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\pool.js","function":null,"line":562,"method":null,"native":false},{"column":11,"file":"G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\pool.js","function":null,"line":995,"method":null,"native":false},{"column":7,"file":"G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\connect.js","function":null,"line":32,"method":null,"native":false},{"column":5,"file":"G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\connect.js","function":"callback","line":280,"method":null,"native":false},{"column":7,"file":"G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\connect.js","function":null,"line":310,"method":null,"native":false},{"column":26,"file":"node:events","function":"Object.onceWrapper","line":485,"method":"onceWrapper","native":false},{"column":20,"file":"node:events","function":"Socket.emit","line":378,"method":"emit","native":false},{"column":8,"file":"node:internal/streams/destroy","function":"emitErrorNT","line":188,"method":null,"native":false},{"column":3,"file":"node:internal/streams/destroy","function":"emitErrorCloseNT","line":153,"method":null,"native":false},{"column":21,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":81,"method":null,"native":false}]}
{"error":{"name":"MongoNetworkError"},"level":"error","message":"uncaughtException: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1137:16) {\n  name: 'MongoNetworkError'\n}]\nMongoNetworkError: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1137:16) {\n  name: 'MongoNetworkError'\n}]\n    at Pool.<anonymous> (G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\topologies\\server.js:438:11)\n    at Pool.emit (node:events:378:20)\n    at G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\pool.js:562:14\n    at G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\pool.js:995:11\n    at G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\connect.js:32:7\n    at callback (G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\connect.js:280:5)\n    at Socket.<anonymous> (G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\connect.js:310:7)\n    at Object.onceWrapper (node:events:485:26)\n    at Socket.emit (node:events:378:20)\n    at emitErrorNT (node:internal/streams/destroy:188:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:153:3)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)","stack":"MongoNetworkError: failed to connect to server [localhost:27017] on first connect [Error: connect ECONNREFUSED 127.0.0.1:27017\n    at TCPConnectWrap.afterConnect [as oncomplete] (node:net:1137:16) {\n  name: 'MongoNetworkError'\n}]\n    at Pool.<anonymous> (G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\topologies\\server.js:438:11)\n    at Pool.emit (node:events:378:20)\n    at G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\pool.js:562:14\n    at G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\pool.js:995:11\n    at G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\connect.js:32:7\n    at callback (G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\connect.js:280:5)\n    at Socket.<anonymous> (G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\connect.js:310:7)\n    at Object.onceWrapper (node:events:485:26)\n    at Socket.emit (node:events:378:20)\n    at emitErrorNT (node:internal/streams/destroy:188:8)\n    at emitErrorCloseNT (node:internal/streams/destroy:153:3)\n    at processTicksAndRejections (node:internal/process/task_queues:81:21)","exception":true,"date":"Sat Feb 06 2021 22:28:21 GMT+0530 (India Standard Time)","process":{"pid":1936,"uid":null,"gid":null,"cwd":"G:\\2021-solution-challenge\\seek","execPath":"C:\\Program Files\\nodejs\\node.exe","version":"v15.8.0","argv":["C:\\Program Files\\nodejs\\node.exe","G:\\2021-solution-challenge\\seek"],"memoryUsage":{"rss":66940928,"heapTotal":57987072,"heapUsed":24071040,"external":20268753,"arrayBuffers":18335759}},"os":{"loadavg":[0,0,0],"uptime":38778},"trace":[{"column":16,"file":"node:net","function":"TCPConnectWrap.afterConnect [as oncomplete]","line":1137,"method":"afterConnect [as oncomplete]","native":false},{"column":11,"file":"G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\topologies\\server.js","function":null,"line":438,"method":null,"native":false},{"column":20,"file":"node:events","function":"Pool.emit","line":378,"method":"emit","native":false},{"column":14,"file":"G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\pool.js","function":null,"line":562,"method":null,"native":false},{"column":11,"file":"G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\pool.js","function":null,"line":995,"method":null,"native":false},{"column":7,"file":"G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\connect.js","function":null,"line":32,"method":null,"native":false},{"column":5,"file":"G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\connect.js","function":"callback","line":280,"method":null,"native":false},{"column":7,"file":"G:\\2021-solution-challenge\\seek\\node_modules\\mongodb\\lib\\core\\connection\\connect.js","function":null,"line":310,"method":null,"native":false},{"column":26,"file":"node:events","function":"Object.onceWrapper","line":485,"method":"onceWrapper","native":false},{"column":20,"file":"node:events","function":"Socket.emit","line":378,"method":"emit","native":false},{"column":8,"file":"node:internal/streams/destroy","function":"emitErrorNT","line":188,"method":null,"native":false},{"column":3,"file":"node:internal/streams/destroy","function":"emitErrorCloseNT","line":153,"method":null,"native":false},{"column":21,"file":"node:internal/process/task_queues","function":"processTicksAndRejections","line":81,"method":null,"native":false}]}
{"level":"error","message":"something failed! something failed!","stack":"Error: something failed!\n    at G:\\2021-solution-challenge\\seek\\routes\\resourcePosts.js:8:11\n    at newFn (G:\\2021-solution-challenge\\seek\\node_modules\\express-async-errors\\index.js:16:20)\n    at Layer.handle [as handle_request] (G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\route.js:137:13)\n    at Route.dispatch (G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\route.js:112:3)\n    at newFn (G:\\2021-solution-challenge\\seek\\node_modules\\express-async-errors\\index.js:16:20)\n    at Layer.handle [as handle_request] (G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\index.js:281:22\n    at param (G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\index.js:354:14)\n    at param (G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\index.js:365:14)\n    at Function.process_params (G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\index.js:410:3)\n    at next (G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\index.js:275:10)\n    at Function.handle (G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\index.js:174:3)\n    at router (G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\index.js:47:12)\n    at newFn (G:\\2021-solution-challenge\\seek\\node_modules\\express-async-errors\\index.js:16:20)\n    at Layer.handle [as handle_request] (G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\layer.js:95:5)"}
{"level":"error","message":"something failed! something failed!","stack":"Error: something failed!\n    at G:\\2021-solution-challenge\\seek\\routes\\resourcePosts.js:8:11\n    at newFn (G:\\2021-solution-challenge\\seek\\node_modules\\express-async-errors\\index.js:16:20)\n    at Layer.handle [as handle_request] (G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at next (G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\route.js:137:13)\n    at Route.dispatch (G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\route.js:112:3)\n    at newFn (G:\\2021-solution-challenge\\seek\\node_modules\\express-async-errors\\index.js:16:20)\n    at Layer.handle [as handle_request] (G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\layer.js:95:5)\n    at G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\index.js:281:22\n    at param (G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\index.js:354:14)\n    at param (G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\index.js:365:14)\n    at Function.process_params (G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\index.js:410:3)\n    at next (G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\index.js:275:10)\n    at Function.handle (G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\index.js:174:3)\n    at router (G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\index.js:47:12)\n    at newFn (G:\\2021-solution-challenge\\seek\\node_modules\\express-async-errors\\index.js:16:20)\n    at Layer.handle [as handle_request] (G:\\2021-solution-challenge\\seek\\node_modules\\express\\lib\\router\\layer.js:95:5)"}
{"stringValue":"\"2983712098371\"","kind":"ObjectId","value":"2983712098371","path":"_id","reason":{},"level":"error","message":"Cast to ObjectId failed for value \"2983712098371\" at path \"_id\" for model \"ResourcePost\" Cast to ObjectId failed for value \"2983712098371\" at path \"_id\" for model \"ResourcePost\"","stack":"CastError: Cast to ObjectId failed for value \"2983712098371\" at path \"_id\" for model \"ResourcePost\"\n    at model.Query.exec (G:\\2021-solution-challenge\\seek\\node_modules\\mongoose\\lib\\query.js:4358:21)\n    at model.Query.Query.then (G:\\2021-solution-challenge\\seek\\node_modules\\mongoose\\lib\\query.js:4452:15)"}
{"stringValue":"\"2983712098371\"","kind":"ObjectId","value":"2983712098371","path":"_id","reason":{},"level":"error","message":"Cast to ObjectId failed for value \"2983712098371\" at path \"_id\" for model \"ResourcePost\" Cast to ObjectId failed for value \"2983712098371\" at path \"_id\" for model \"ResourcePost\"","stack":"CastError: Cast to ObjectId failed for value \"2983712098371\" at path \"_id\" for model \"ResourcePost\"\n    at model.Query.exec (G:\\2021-solution-challenge\\seek\\node_modules\\mongoose\\lib\\query.js:4358:21)\n    at model.Query.Query.then (G:\\2021-solution-challenge\\seek\\node_modules\\mongoose\\lib\\query.js:4452:15)"}
